payroll-processor-functions/src/payroll-processor-functions/Features/Employees/EmployeesTrigger.cs

```diff
@@ -0,0 +1,52 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using System.Threading.Tasks;
+using PayrollProcessor.Functions.Infrastructure;
+using Microsoft.WindowsAzure.Storage.Table;
+using PayrollProcessor.Functions.Features.Resources;
+using System;
+using System.Linq;
+
+namespace PayrollProcessor.Functions.Features.Employees
+{
+    public class EmployeesTrigger
+    {
+        [FunctionName(nameof(GetEmployees))]
+        public async Task<ActionResult<Employee[]>> GetEmployees(
```

Comment:
The way `ActionResult<T>` is defined, we can't return an `ActionResult<IEnumerable<T>>` here, so anywhere we'd be returning interfaces, they need to be converted to concrete types.

In the collection of `IEnumerable<T>`, I've turned them all into arrays.